cpp/models/lct_secir/parameters.h

```diff
@@ -135,10 +136,11 @@ struct TransmissionProbabilityOnContact {
 struct ContactPatterns {
     using Type = UncertainContactMatrix<ScalarType>;
 
-    static Type get_default()
+    static Type get_default(size_t size)
     {
-        ContactMatrixGroup contact_matrix = ContactMatrixGroup(1, 1);
-        contact_matrix[0]                 = mio::ContactMatrix(Eigen::MatrixXd::Constant(1, 1, 10.));
+        Eigen::Index size_idx = (Eigen::Index)size;
```

Comment:
Why? For type conversion?

Reply:
Yes, but maybe it is more efficient to do the type conversion where needed.